### ISSUE TITLE
BV calculator fault

### ISSUE THREAD

**Issue by jbhuddleston** [MegaMek/mekhq#1169]
### Environment
MekHQ 0.45.4, Windows 7 Ultimate, Java version 1.8.0_201

### Description
1. BattleValue of opposing forces calculates incorrectly for a scenario.
2. Opposing forces are extremely strong, generally 3 or 4 to 1. Maybe not a bug but it happened in two scenarios the first campaign attempt and again on the second attempt after disallowing anything but meks. In both cases I had a lance of modified Firestarters on a Raid mission. Previous camapigns in 0.45.3 were almost evenly matched.

### Files
![8CH](https://user-images.githubusercontent.com/49171387/55364200-1be0a900-54b6-11e9-95ba-3a6664a0cf3f.PNG)
![8CH2](https://user-images.githubusercontent.com/49171387/55364207-23a04d80-54b6-11e9-918c-0652406bef4d.PNG)
[8th Canadian Hussars.zip](https://github.com/MegaMek/mekhq/files/3031644/8th.Canadian.Hussars.zip)
[8CH.zip](https://github.com/MegaMek/mekhq/files/3031636/8CH.zip)
[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/3031637/mekhqlog.txt)

**Comment by VicenteCartas** [comment]
This is a duplicate of the issue fixed by commit #1163

**Comment by jbhuddleston** [comment]
Apologies for submitting a duplicate issue regarding point 1.
Is point 2 a result of point 1, just an unhappy coincidence or a result of some other setting or issue?

**Comment by VicenteCartas** [comment]
So, the BV calculations are off by a lot, but that wouldn't affect force composition. Sounds more like a settings issue to me.

**Comment by jbhuddleston** [comment]
Thanks. I will try a few other campaign setups to see what I might have
done.

**Comment by NickAragua** [comment]
Other than the traditional "git gud", you can drop the AtB difficulty setting down to green or ultra-green and uncheck "2x vehicles".

**Comment by VicenteCartas** [comment]
If you have same setup configs in two MekHQ versions and you get wildly different forces, then please open a bug with it and I'll give it a look.

**Comment by jbhuddleston** [comment]
Sorry, new to the lingo. "git gud"?
Already unchecked 2x vehicles and believe I played an easier few scenarios
on Regular. (Skill Level 2 in the mmconf\mhqPresets files I have been
using.) Will try Green.

I did manually set the bot.preferences to
forceIndividualInitiative=true and that may have put the difficulty=3 into
effect, but I think that just affects bot performance, not force generation.
